code/pom.xml

```diff
@@ -58,6 +58,16 @@
       <artifactId>google-api-services-people</artifactId>
       <version>v1-rev537-1.25.0</version>
     </dependency>
+    <dependency>
+    <groupId>com.google.maps</groupId>
+      <artifactId>google-maps-services</artifactId>
+      <version>0.14.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
```

Comment:
What is this for? Why do we need it? This looks not very trustwrorthy.

Reply:
I don't see it used anywhere.

Reply:
It is from the github repo:
https://github.com/googlemaps/google-maps-services-java

Reply:
👍